src/block/request.rs

```diff
@@ -49,6 +49,8 @@ const VIRTIO_BLK_T_FLUSH: u32 = 4;
 const VIRTIO_BLK_T_DISCARD: u32 = 11;
 const VIRTIO_BLK_T_WRITE_ZEROES: u32 = 13;
 
+const VIRTIO_BLK_DISCARD_WRITE_ZEROES_FLAG_UNMAP: u32 = 1;
```

Comment:
nit: could you call this `VIRTIO_BLK_WRITE_ZEROES_FLAG_UNMAP` instead to be consistent with the [kernel name](https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/virtio_blk.h#L178)? Makes stuff easier to find in the driver code